profiles/common/modules/custom/multisite_config/lib/Drupal/comment/Config.php

```diff
@@ -46,7 +58,95 @@ public function setDefaultCommentForContentType($content_type, $value) {
    *   Value to be set (boolean).
    */
   public function setThreadingCommentForContentType($content_type, $value) {
-    variable_set('comment_form_location_' . $content_type, $value);
+    variable_set('comment_default_mode_' . $content_type, $value);
+  }
+
+  /**
+   * Set title comment setting for a specific content type.
+   *
+   * @param string $content_type
+   *   Content type machine name.
+   * @param string $value
+   *   Value to be set (boolean).
+   */
+  public function setTitleCommentForContentType($content_type, $value) {
+    variable_set('comment_subject_field_' . $content_type, $value);
+  }
+
+  /**
+   * Set preview comment setting for a specific content type.
+   *
+   * @param string $content_type
+   *   Content type machine name.
+   * @param string $value
+   *   Value to be set (human readable).
+   */
+  public function setPreviewCommentForContentType($content_type, $value) {
+    switch ($value) {
+      case 'required':
+        $value_id = 2;
+        break;
+
+      case 'optional':
+      default:
+        $value_id = 1;
+        break;
+
+      case 'disabled':
+        $value_id = 0;
+        break;
+    }
+    variable_set('comment_preview_' . $content_type, $value_id);
+  }
+
+  /**
+   * Set comments per page setting for a specific content type.
+   *
+   * @param string $content_type
+   *   Content type machine name.
+   * @param string $value
+   *   Value to be set (human readable).
+   */
+  public function setNumberCommentForContentType($content_type, $value) {
+    switch ($value) {
```

Comment:
Where these cases come from, are they random? The ticket says to remove the default 50, but nothing regarding the options.

Reply:
These cases have been taken from the options given in the select field for the number of comments in a content type. You can check it for example in /admin/structure/types/manage/article_en.
As the given options are 10, 30, 50, 70, 90, 150, 200, 250 and 300, these are also the cases that I have added to  the function.